tests/data/tag_windows.py

```diff
@@ -168,6 +169,18 @@ def testApplicationExecution(self):
         winjob.WinJobEventData, attribute_values_per_name,
         ['application_execution'])
 
+    # Test: parser is 'winreg/amcache' AND
+    #       data_type is 'windows:registry:key_value' AND
+    #       key_path contains 'InventoryApplicationFile\\'
+    event = events.EventObject()
+    event.timestamp = self._TEST_TIMESTAMP
+    event.timestamp_desc = definitions.TIME_DESCRIPTION_MODIFICATION
+    event_data = windows_events.WindowsRegistryEventData()
+    event_data.key_path = '\\Root\\InventoryApplicationFile\\7z.exe|afe683e0fa522625'
+    event_data.parser = 'winreg/amcache'
+    storage_writer = self._TagEvent(event, event_data, None)
+    self._CheckLabels(storage_writer, ['application_execution'])
```

Comment:
Why not use `_CheckTaggingRule` ?

Reply:
No real reason. I saw it written like this in other tests. Should I change it to `_CheckTaggingRule`?

Reply:
yes, please, unless there is a specific reason not to, also see my comment about AMCacheFileEventData. The current filter rule is fragile since it tests a path as a string, not as a path using using `PATH()`